onnx/defs/schema.h

```diff
@@ -456,23 +471,26 @@ class OpSchemaRegistry {
       }
       auto& m = map();
       auto& key = op_schema.Name();
-      if (m.count(key)) {
-        const auto& schema = m[key];
+      auto ver = op_schema.SinceVersion();
+
+      if (m[key].count(ver)) {
+        const auto& schema = m[key][ver];
         std::cerr << "Trying to register schema with name " << key
-                  << " from file " << op_schema.file() << " line "
+                  << " (version " << ver << ") from file " << op_schema.file() << " line "
                   << op_schema.line()
                   << ", but it is already registered from file "
                   << schema.file() << " line " << schema.line();
         abort();
       }
-      m.emplace(std::make_pair(key, op_schema));
+      m[key].emplace(std::make_pair(ver, op_schema));
     }
   };
 
+  // Return the latest schema for an operator
   static const OpSchema* Schema(const std::string& key) {
```

Comment:
Change the API a little bit as "const OpSchema* LatestSchemaSinceVersion(const std::string& key, int version = MAX_INT)", so that backend will be able to call this API to get the lowest version it wants to support.

Reply:
This is a good idea! I wonder if we shouldn't also implement this for the "get all operators" function.